app/app.ts

```diff
@@ -1,2 +1,11 @@
-﻿import * as app from 'application';
-app.start({ moduleName: "main-page" });
+﻿import * as application from 'application';
+import {TNSFontIcon, fonticon} from 'nativescript-fonticon';
+
+TNSFontIcon.debug = true;
+TNSFontIcon.paths = {
+  'fa': 'font-awesome.css'
+};
+TNSFontIcon.loadCss();
+
+application.resources['fonticon'] = fonticon;
```

Comment:
@tjvantoll no those are just for demo purposes. Although... I think I did bake the `fa` it into the app.css that should be pulled out and set per component on second thought.